packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/observe/TransactionsTable.tsx

```diff
@@ -97,6 +97,7 @@ export function TransactionsTable() {
     state: {
       expanded,
     },
+    getRowId: (row) => row.hash || row.writeId,
```

Comment:
might consider `writeId` first since its more stable so that the ID doesn't change once the hash appears?

Reply:
I'd lean towards using `writeId || hash` just because it already creates a stable identity for the row. Also, changed the order but somehow can't quite get typescript to infer that either of these types will be non-null, strange that it works in the opposite order. I wonder given the type coercion `row.writeId || row.hash!` if the discriminated union is still necessary, and could just do `hash?: hex; writeId?: string` for type.

Reply:
Changed to using `writeId` as the main key. `writeId` is set to `hash` when tx comes from blocks watcher.